client/src/App.jsx

```diff
@@ -14,7 +14,7 @@ import { UserContext } from './context/UserContext'
 import Profile from './views/Profile';
 import axios from 'axios';
 import AdminUserList from './views/AdminUserList';
-import AdminDashboard from './views/AdminDashboard';
+import AdminDashboard from './components/AdminDashboard';
```

Comment:
deleted ./views/AdminDashboard - not needed.
KEEP the correct version of AdminDashboard  in ./components/AdminDashboard